RequestGroup/RequestGroup-MedicationsPrescription-Example.xml

```diff
@@ -0,0 +1,189 @@
+<RequestGroup xmlns="http://hl7.org/fhir">
+  <id value="RequestGroup-MedicationsPrescription-Example" />
+  <meta>
+    <versionId value="2" />
+    <lastUpdated value="2025-01-30T15:43:00+00:00" />
+  </meta>
+  <extension url="https://fhir.nhs.uk/StructureDefinition/Extension-EPS-RepeatInformation">
+    <extension url="numberOfRepeatsAllowed">
+      <valueInteger value="5" />
+    </extension>
+    <extension url="numberOfRepeatsIssued">
+      <valueInteger value="2" />
+    </extension>
+  </extension>
+  <extension url="https://fhir.nhs.uk/StructureDefinition/Extension-PendingCancellation">
+    <valueBoolean value="false" />
+  </extension>
+  <extension url="https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionStatusHistory">
+    <extension url="status">
+      <valueCoding>
+        <system value="https://fhir.nhs.uk/CodeSystem/EPS-task-business-status" />
+        <code value="0003" />
+        <display value="With Dispenser - Active" />
+      </valueCoding>
+    </extension>
+  </extension>
+  <extension url="https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionType">
+    <valueCoding>
+      <system value="https://fhir.nhs.uk/CodeSystem/prescription-type" />
+      <code value="0101" />
+      <display value="Primary Care Prescriber - Medical Prescriber" />
+    </valueCoding>
+  </extension>
+  <identifier>
+    <system value="https://fhir.nhs.uk/Id/prescription-order-number" />
+    <value value="269CA9-A99968-4577BP" />
+  </identifier>
+  <status value="active" />
+  <intent value="reflex-order" />
+  <subject>
+    <reference value="Patient/Patient-StaceyTwitchett-Example" />
+    <identifier>
+      <system value="https://fhir.nhs.uk/Id/nhs-number" />
+      <value value="5839945242" />
+    </identifier>
+  </subject>
+  <authoredOn value="2025-01-29T12:00:00Z" />
+  <author>
+    <reference value="Organization/UKCore-Organization-HallgarthSurgery-Example" />
```

Comment:
Need to check with EPS team whether this should be a practitioner/practitionerRole instead

Reply:
Changed to Practitioner